### DISCUSSION TITLE
Cannot setup Photos folder

### DISCUSSION THREAD

**Discussion by kindlydodo** [photoview/photoview#381] [Q&A and Troubleshooting]
All i want is to setup photoview, and set my Image Directory to : "/home/user/tmp/testimages". I followed the README.md, but it dont work.

That is what i did:

1. git clone
2. created the docker-compose.yml file and edit the file like described in de comment: add a line at 48:
    - ./photos_path:/photos:ro
    - /home/user/tmp/testimages:/photos:ro
3. sudo docker-compose up -d

Now, i open localhost:8000, add a user, but there are no folders / images under "photos" / "albums"

what am I doing wrong?

additionally:

At the "Initial Setup", on the "initial setup wizard" when i add a username and password: What are the "Photo Phat" input Box for? I expect to add the Path of my Image Directory, so i setup a fresh Docker instance, whithout customize the docker-compose.yml file (like described above), and put my path inside: "/home/user/tmp/testimages"...But, i get an error

**Comment by viktorstrate** [comment]
Make sure you don't have two volume mounts for `/photos` (delete `./photos_path:/photos:ro` but keep `/home/user/tmp/testimages:/photos:ro`.

> and put my path inside: "/home/user/tmp/testimages"...But, i get an error

Because you are using Docker, Photoview does not have direct access to your file system, that's why you added the line under volumes. You should write `/photos` for the root path.

For more info, take a look at Docker bind mounts https://docs.docker.com/storage/bind-mounts/

**Comment by 0lsn** [comment]
Hey, I got a similar problem. I cant mount "/mnt/w/photos" but I can mount "/mnt/". Am I doing something wrong?
I am using latest docker, wsl 2.0 with Kali-Linux on the latest W10

![2021_06_19_17-03-29_Code](https://user-images.githubusercontent.com/24460744/122646612-4a431100-d120-11eb-9e2d-58ab30860f9f.png)

![2021_06_19_17-08-25_Docker_Desktop](https://user-images.githubusercontent.com/24460744/122646736-0ac8f480-d121-11eb-855c-9cf95c940c4c.png)


Editing the yml file to the following path > "/mnt"

![2021_06_19_17-11-12_msedge](https://user-images.githubusercontent.com/24460744/122646806-64312380-d121-11eb-8761-93440b70dd8d.png)

**Reply by viktorstrate** [reply]
`/mnt/w/photos:/photos:ro` binds `/mnt/w/photos` from the host machine to `/photos` within the docker container. That means that you should add `/photos` as the photo path on the website instead of `/mnt`.
